TestAssets/TestProjects/PerformanceTestProjects/dir.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0">
```

Comment:
Do we really need this file? Is it because these tests still can't run on the newest SDK? We need to pursue fixing that bug in the SDK, if that's the case.